### DISCUSSION TITLE
GCAD (Global Context Anomaly Detection)

### DISCUSSION THREAD

**Discussion by alexriedel1** [open-edge-platform/anomalib#719] [Feature Requests]
The method and the accompanying dataset can be found here:
https://link.springer.com/content/pdf/10.1007/s11263-022-01578-9.pdf

The method allows to also find "logical" anomalies in an image
![image](https://user-images.githubusercontent.com/54716527/201100447-3388033f-e5f6-4314-a169-37a1d1a42602.png)

**Comment by samet-akcay** [comment]
Thanks @alexriedel1, it's on our todo list. I'll move this to feature requests.
